src/loaders/ImageLoader.js

```diff
@@ -39,11 +39,32 @@ Object.assign( ImageLoader.prototype, {
 
 			var loader = new XHRLoader();
 			loader.setPath( this.path );
-			loader.setResponseType( 'blob' );
+			loader.setResponseType( undefined !== this.responseType ? this.responseType : 'blob' );
 			loader.setWithCredentials( this.withCredentials );
-			loader.load( url, function ( blob ) {
-
-				image.src = URL.createObjectURL( blob );
+			loader.load( url, function ( response ) {
+
+				if (loader.responseType == THREE.ResponseType.Blob) {
```

Comment:
As it is something available for the users, I thought that using auto-completable predefined values would be a good idea.